modules/contact/src/constraints/ComputeWeightedGapLMMechanicalContact.C

```diff
@@ -146,25 +166,47 @@ ComputeWeightedGapLMMechanicalContact::computeJacobian(const Moose::MortarType m
 void
 ComputeWeightedGapLMMechanicalContact::post()
 {
-  for (const auto & pr : _node_to_weighted_gap)
+  for (const auto & pr : _dof_to_weighted_gap)
   {
-    _weighted_gap_ptr = &pr.second;
-    enforceConstraintOnNode(pr.first);
+    _weighted_gap_ptr = &pr.second.first;
+    _normalization_ptr = &pr.second.second;
+
+    enforceConstraintOnDof(pr.first);
   }
 }
 
 void
-ComputeWeightedGapLMMechanicalContact::enforceConstraintOnNode(const Node * const node)
+ComputeWeightedGapLMMechanicalContact::wrongPost(const std::unordered_set<const Node *> & inactive_lm_nodes)
```

Comment:
`wrongPost`, could be more descriptive.

Reply:
It's a copy of post that skips assembling any inactive nodes when the old edge dropping behavior is enabled, I was just trying to keep with naming these things like their counterpart routines but with `wrong` to point out they correspond to the `give_me_wrong_results` option. 

I can try to think of something more descriptive but they're supposed to be deprecated soon anyway